proxysql/build-binary-proxysql

```diff
@@ -12,7 +12,16 @@ set -o xtrace
 if [ -f /usr/bin/yum ]; then
     RHEL=$(rpm --eval %rhel)
     if [[ $RHEL -eq 8 ]]; then
-        sudo yum -y install cmake python2 gnutls-devel libtool || true
+        sudo yum -y install python2 gnutls-devel libtool || true
+        sudo yum remove -y cmake
+        sudo wget https://github.com/Kitware/CMake/releases/download/v3.19.8/cmake-3.19.8.tar.gz
+        sudo tar -zxvf cmake-3.19.8.tar.gz
+        cd cmake-3.19.8/
+        sudo ./bootstrap
+        sudo make
```

Comment:
Why not club both lines together
sudo make install

Reply:
Such commands sequence are taken from the official cmake site.
<img width="943" alt="Screenshot 2022-06-16 at 18 08 12" src="https://user-images.githubusercontent.com/25184101/174101483-7aa544cd-207e-428b-b2a6-5eda2556a59c.png">